search-website/README.md

```diff
@@ -0,0 +1,123 @@
+# Azure Cognitive Search UI
+
+This sample is a React template for [Azure Cognitive Search](https://docs.microsoft.com/en-us/azure/search/search-what-is-azure-search). It leverages the [Azure SDK for Javascript/Typescript](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/search/search-documents/) and [Azure Static Web Apps](https://aka.ms/swadocs) to make it easy to get up and running with a simple web application.
+
+This README is an shortened version of the [full tutorial](https://aka.ms/search-website-tutorial). 
+
+Related resources: 
+* [Demo](https://victorious-beach-0ab88b51e.azurestaticapps.net/)
+* [Tutorial](https://aka.ms/search-website-tutorial)
+
+![Screenshot of sample web app](./images/web-app.png)
+
+You can easily deploy the sample onto Azure or run it locally by following the steps below.
+
+## Running the application locally
+
+To run the sample locally, follow the steps below.
+
+### Prerequisites
+
+- A GitHub account
+- [Node.js and Git](https://nodejs.org/)
+- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=shopathome-github-jopapa) installed
+- The [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions&WT.mc_id=shopathome-github-jopapa) installed
+- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=shopathome-github-jopapa) installed
+
+### Setup
+
+1. Clone (or Fork and Clone) this repository
+1. Open `./search-website` in Visual Studio Code. 
+1. Rename the `api/local.settings.json.rename` file to `api/local.settings.json`.
+
+    The `local.settings.json` file holds all of the keys that the application needs.
+    
+    For this sample, there is a search index using the [goodbooks-10k dataset](https://github.com/zygmuntz/goodbooks-10k) that that is publicly available using the connection information below. The index consists of 10,000 popular books that we'll search over in our application.
+    
+    ```json
+    {
+      "IsEncrypted": false,
+      "Values": {
+        "AzureWebJobsStorage": "",
+        "FUNCTIONS_WORKER_RUNTIME": "node",
+        "SearchApiKey": "REPLACE-WITH-YOUR-SEARCH-QUERY-KEY",
+        "SearchServiceName": "REPLACE-WITH-YOUR-SEARCH-RESOURCE-NAME",
+        "SearchIndexName": "good-books",
+        "SearchFacets": "authors*,language_code"
+      }
+    }
+    ```
+
+## Run the app locally
+
+This project can be run anywhere, but VS Code is required for local debugging.
+
+1. Open the `./search-website` application with VS Code.
+
+### Running the front-end
+
+1. From a terminal in the `./search-website` root, install front-end dependencies...
+
+   ```bash
+   npm install
```

Comment:
I think this step requires path information.  I ran it, but it says "no such file or directory, open 'C:\Users\heidist\package.json'"   For testing, I put the folder off my C root drive, not my users drive.  I'll continue testing, but I think i'm probably not in good shape at this point.